src/js/menu/menu.js

```diff
@@ -122,6 +169,43 @@ class Menu extends Component {
     }
   }
 
+  /**
+   * Called when a `MenuItem` gets clicked or tapped.
+   *
+   * @param {EventTarget~Event} event
+   *        The `click` or `tap` event that caused this function to be called.
+   *
+   * @listens click,tap
+   */
+  handleTapClick(event) {
+    // Unpress the associated MenuButton, and move focus back to it
+    if (this.menuButton_) {
+      this.menuButton_.unpressButton();
+
+      const childComponents = this.children();
+
+      if (!Array.isArray(childComponents)) {
+        return;
+      }
+
+      const foundComponent = childComponents.filter(component => {
+        if (component.el() === event.target) {
```

Comment:
since filter expects us to return a boolean, we don't need the if statement here, we can just return the value of this comparison.

Reply:
Oh...I don't think about it. thanks.